saleor/tests/fixtures.py

```diff
@@ -818,11 +818,9 @@ def graphql_address_data():
         "lastName": "Doe Mirumee",
         "companyName": "Mirumee Software",
         "streetAddress1": "Tęczowa 7",
-        "streetAddress2": "",
```

Comment:
Do we have to remove these fields? I would keep them; they can be empty (at least for PL) so they should be valid data.